travis.sh

```diff
@@ -15,6 +15,7 @@ function strongEcho {
   echo "================ $1 ================="
 }
 
+build_snapshot "SonarSource/sonarlint-core"
```

Comment:
Please use a QA build of sonarlint-core before merging (in fact I think we should stop using SNAPSHOTs except locally).